pages/common/histexpand.md

```diff
@@ -3,10 +3,6 @@
 > Reuse and expand the shell history in `sh`, `bash`, `zsh`, `rbash` and `ksh`.
 > More information: <https://www.gnu.org/software/bash/manual/html_node/History-Interaction>.
 
-- Run the previous command:
-
-`!!`
```

Comment:
Despite the fact that this example is simple, I think it's one of the most important on the page, as it introduces the user to the concept of history expansion with something they may already be familiar with:

![](https://imgs.xkcd.com/comics/sandwich.png)

Reply:
It's sometimes hard to keep in mind for whom tldr is indented, so I have to agree. I think it would be best if we would make a poll, if and which other command to remove.

(BTW, how do some people have the ability to always find the right xkcd in the right moment?)

Reply:
\* indented → intended

I'd prefer no polls here - generally speaking we try to talk it out instead, so everyone's happy with the outcome. @ waldyrious explained this in a previous issue/PR a while back

Reply:
Makes sense. So, how if we romevo one of those:
* `!-n` - I know I wanted to keep it at the beginning but I'm really sure that it's really useful anymore
* One of either `!^` or `!$`, since I think they're both rarely used. Maybe we could even combine them somehow.

Reply:
What about removing `!^`? `!$` can be useful  e.g. in this situation:

```
nano path/to/file
node !$
```

Reply:
Any other opinions? @bl-ue, @navarroaxel, @CleanMachine1

Reply:
I think we can merge `!!` and `sudo !!` but otherwise remove `!^` is a good option.

Reply:
I think we can merge `!!` and `sudo !!` but otherwise remove `!^` is a good option.

Reply:
That's also an option. If we merged the first to into
```
- Run the previous command as root (`!!` replaces the previous command):

sudo !!
```
we'd solve the problem while still explaining the basic function of `!!` and not deleting any example.

Reply:
Ah, that works @navarroaxel @marchersimon - let's do that.

Reply:
@jopefd could you apply this change?

Reply:
I made a [suggestion](https://github.com/tldr-pages/tldr/pull/6051/files#r657176121) for that